test/unit/reductions/exponentiated_gradient/test_utilities.py

```diff
@@ -5,15 +5,22 @@
 import pandas as pd
 
 sensitive_features = [str(x) for x in 'AAAAAAA' 'BBBBBBB' 'CCCCCC']
-labels =     [int(x) for x in '0110100' '0010111' '001111']  # noqa: E222
-X1 =         [int(x) for x in '0110101' '0111101' '001011']  # noqa: E222
-X2 =         [int(x) for x in '0000100' '0000011' '111111']  # noqa: E222
-X3 =         [int(x) for x in '1111111' '1111111' '111111']  # noqa: E222
+labels =             [int(x) for x in '0110100' '0010111' '001111']  # noqa: E222
+X1 =                 [int(x) for x in '0110101' '0111101' '001011']  # noqa: E222
+X2 =                 [int(x) for x in '0000100' '0000011' '111111']  # noqa: E222
+X3 =                 [int(x) for x in '1111111' '1111111' '111111']  # noqa: E222
+scores =          [int(x)/10 for x in '0001124' '0223346' '134447']  # noqa: E222
```

Comment:
this looks buggy, you probably mean `float(x)/10` or `int(x)/10.0`?

Reply:
Nope. 
```
>>> int(7)/10
0.7
```

Reply:
It's all identical.